test/fixtures/rules/std_rule_cases/L039.yml

```diff
@@ -112,3 +112,93 @@ test_redshift_at_time_zone:
   configs:
     core:
       dialect: redshift
+
+test_excess_space_without_align_alias:
+  fail_str: |
+        SELECT
+            a    AS first_column,
+            b      AS second_column,
+            (a + b) / 2 AS third_column
+        FROM foo
+  fix_str: |
+    SELECT
+        a AS first_column,
+        b AS second_column,
+        (a + b) / 2 AS third_column
+    FROM foo
+  configs:
+    rules:
+      L039:
+        align_alias: false
+
+test_excess_space_with_align_alias:
+  fail_str: |
+        SELECT
+            a    AS first_column,
+            b      AS second_column,
+            (a + b) / 2 AS third_column
+        FROM foo
+  fix_str: |
+    SELECT
+        a           AS first_column,
+        b           AS second_column,
+        (a + b) / 2 AS third_column
+    FROM foo
+  configs:
+    rules:
+      L039:
+        align_alias: true
+
+test_missing_keyword_with_align_alias:
+  fail_str: |
+        SELECT
+            a    first_column,
+            b      AS second_column,
+            (a + b) / 2 AS third_column
+        FROM foo
+  fix_str: |
+    SELECT
+        a            first_column,
```

Comment:
This looks strange -- `first_column` is not aligned with `second_column`.

Reply:
The reason I wrote this `fix_str` result is because `L039` only handles the whitespace padding, and `L011` adds the 3-char str `AS ` (through L011's rule config `aliasing: explicit`)

I wanted to include `L011` during the test, but couldn't find any examples of applying multiple rules during a unit test.

Is there a way to leave a description of a test in the yml file, or do you have any thoughts on what we can do to clean this up?

Reply:
I thought `main` was a few commits ahead of this branch and re-added the `pragma: no cover` a few commits back, `c78f923`.

The head of branch, `f81e0ee`, re-removes `pramga: no cover` as per #3808

Reply:
The thing that confuses me is, it's changing the whitespace, but not actually aligning it with the others, and it's off by only one space. Doesn't that imply that if `AS` were added, it'd still be indented wrong?

You can create tests that run multiple rules. Those work a bit differently. See the test cases in `test/fixtures/linter/autofix/ansi/` (also other adjacent directories for other dialects). There is one directory per test, and you specify the rules to run in `test-config.yml`.

Reply:
There's a file `test/fixtures/linter/autofix/README.md` which explains these tests in more detail.

Reply:
I'm lazy, so I usually begin by copying an entire directory (i.e. one of the other test cases). For me, it's easier to **change** a test than to create a new one.